diff -uNr nant-0.92.old/Makefile nant-0.92/Makefile
--- nant-0.92.old/Makefile	2015-05-12 22:23:37.390417588 +0200
+++ nant-0.92/Makefile	2015-05-12 22:25:49.997066282 +0200
@@ -85,7 +85,7 @@
 	$(NANT) $(TARGET_FRAMEWORK) -f:NAnt.build test
 	
 bootstrap/NAnt.exe:
-	$(MCS) $(DEBUG) -target:exe -define:$(DEFINE) -out:bootstrap${DIRSEP}NAnt.exe -r:bootstrap${DIRSEP}log4net.dll \
+	$(MCS) $(DEBUG) -target:exe -define:$(DEFINE) -out:bootstrap${DIRSEP}NAnt.exe -pkg:log4net \
 		-r:System.Configuration.dll -recurse:src${DIRSEP}NAnt.Console${DIRSEP}*.cs src${DIRSEP}CommonAssemblyInfo.cs
 
 
@@ -95,15 +95,12 @@
 setup:
 	mkdir -p bootstrap
 	cp -R lib/ bootstrap/lib
-	# Mono loads log4net before privatebinpath is set-up, so we need this in the same directory
-	# as NAnt.exe
-	cp lib/common/neutral/log4net.dll bootstrap
 	cp src/NAnt.Console/App.config bootstrap/NAnt.exe.config
 
 bootstrap/NAnt.Core.dll:
 	$(RESGEN)  src/NAnt.Core/Resources/Strings.resx bootstrap/NAnt.Core.Resources.Strings.resources
 	$(MCS) $(DEBUG) -target:library -warn:0 -define:$(DEFINE) -out:bootstrap/NAnt.Core.dll -debug \
-		-resource:bootstrap/NAnt.Core.Resources.Strings.resources -r:lib${DIRSEP}common${DIRSEP}neutral${DIRSEP}log4net.dll \
+		-resource:bootstrap/NAnt.Core.Resources.Strings.resources -pkg:log4net \
 		-r:System.Web.dll -recurse:src${DIRSEP}NAnt.Core${DIRSEP}*.cs src${DIRSEP}CommonAssemblyInfo.cs
 
 bootstrap/NAnt.DotNetTasks.dll:
diff -uNr nant-0.92.old/src/NAnt.Console/NAnt.Console.build nant-0.92/src/NAnt.Console/NAnt.Console.build
--- nant-0.92.old/src/NAnt.Console/NAnt.Console.build	2012-06-09 16:05:53.000000000 +0200
+++ nant-0.92/src/NAnt.Console/NAnt.Console.build	2015-05-12 22:24:11.214838158 +0200
@@ -32,10 +32,12 @@
                 <include name="../CommonAssemblyInfo.cs" />
             </sources>
             <references>
-                <include name="${build.dir}/bin/log4net.dll" />
                 <include name="System.Configuration.dll" />
                 <include name="System.Xml.dll" />
             </references>
+            <pkg-references>
+                <package name="log4net"/>
+            </pkg-references>
             <resources>
                 <include name="*.resx"/>
             </resources>
diff -uNr nant-0.92.old/src/NAnt.Core/NAnt.Core.build nant-0.92/src/NAnt.Core/NAnt.Core.build
--- nant-0.92.old/src/NAnt.Core/NAnt.Core.build	2012-06-09 16:05:53.000000000 +0200
+++ nant-0.92/src/NAnt.Core/NAnt.Core.build	2015-05-12 22:24:11.215838170 +0200
@@ -27,9 +27,11 @@
                 <include name="Resources/**/*" />
             </resources>
             <references>
-                <include name="${build.dir}/bin/log4net.dll"/>
                 <include name="System.Web.dll"/>
             </references>
+            <pkg-references>
+                <package name="log4net"/>
+            </pkg-references>
         </csc>
     </target>
 </project>
