diff -uNr nant-0.92.old/Makefile nant-0.92/Makefile
--- nant-0.92.old/Makefile	2012-06-09 16:05:52.000000000 +0200
+++ nant-0.92/Makefile	2015-05-12 22:43:12.390025959 +0200
@@ -109,7 +109,7 @@
 bootstrap/NAnt.DotNetTasks.dll:
 	$(RESGEN)  src/NAnt.DotNet/Resources/Strings.resx bootstrap/NAnt.DotNet.Resources.Strings.resources
 	$(MCS) $(DEBUG) -target:library -warn:0 -define:$(DEFINE) -out:bootstrap/NAnt.DotNetTasks.dll \
-		-r:./bootstrap/NAnt.Core.dll -r:bootstrap/lib/common/neutral/NDoc.Core.dll \
+		-r:./bootstrap/NAnt.Core.dll \
 		-recurse:src${DIRSEP}NAnt.DotNet${DIRSEP}*.cs -resource:bootstrap/NAnt.DotNet.Resources.Strings.resources \
 		src${DIRSEP}CommonAssemblyInfo.cs
 
diff -uNr nant-0.92.old/src/NAnt.DotNet/NAnt.DotNet.build nant-0.92/src/NAnt.DotNet/NAnt.DotNet.build
--- nant-0.92.old/src/NAnt.DotNet/NAnt.DotNet.build	2012-06-09 16:05:53.000000000 +0200
+++ nant-0.92/src/NAnt.DotNet/NAnt.DotNet.build	2015-05-12 22:43:12.390025959 +0200
@@ -22,8 +22,6 @@
             </sources>
             <references>
                 <include name="${build.dir}/bin/NAnt.Core.dll" />
-                <include name="${nant::scan-probing-paths(build.dir + '/bin','NDoc.ExtendedUI.dll')}" />
-                <include name="${nant::scan-probing-paths(build.dir + '/bin','NDoc.Core.dll')}" />
             </references>
             <resources prefix="NAnt.DotNet" dynamicprefix="true">
                 <include name="Resources/**/*" />
diff -uNr nant-0.92.old/src/NDoc.Documenter.NAnt/NDoc.Documenter.NAnt.build nant-0.92/src/NDoc.Documenter.NAnt/NDoc.Documenter.NAnt.build
--- nant-0.92.old/src/NDoc.Documenter.NAnt/NDoc.Documenter.NAnt.build	2012-06-09 16:05:55.000000000 +0200
+++ nant-0.92/src/NDoc.Documenter.NAnt/NDoc.Documenter.NAnt.build	2015-05-12 22:44:20.867890861 +0200
@@ -23,8 +23,6 @@
             </sources>
             <references basedir="${build.dir}/bin" failonempty="true">
                 <include name="NAnt.Core.dll" />
-                <include name="${nant::scan-probing-paths('NDoc.ExtendedUI.dll')}" />
-                <include name="${nant::scan-probing-paths('NDoc.Core.dll')}" />
                 <include name="System.Web.dll" />
             </references>
             <resources basedir="Resources" prefix="Documenter.xslt.">
