--- a/src/nunit-gui/nunit-gui.csproj	2016-10-08 23:29:53.099752797 +0200
+++ b/src/nunit-gui/nunit-gui.csproj	2016-10-08 23:32:57.909364771 +0200
@@ -37,20 +37,20 @@
       <HintPath>..\..\packages\NUnit.Engine.3.4.1\lib\Mono.Cecil.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit-agent, Version=3.4.1.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.4.1\lib\nunit-agent.exe</HintPath>
+    <Reference Include="nunit-agent">
+      <HintPath>/usr/lib/mono/nunit/nunit-agent.exe</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit-agent-x86, Version=3.4.1.0, Culture=neutral, processorArchitecture=x86">
-      <HintPath>..\..\packages\NUnit.Engine.3.4.1\lib\nunit-agent-x86.exe</HintPath>
+    <Reference Include="nunit-agent-x86">
+      <HintPath>/usr/lib/mono/nunit/nunit-agent-x86.exe</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit.engine, Version=3.4.1.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.4.1\lib\nunit.engine.dll</HintPath>
+    <Reference Include="nunit.engine">
+      <HintPath>/usr/lib/mono/nunit/nunit.engine.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit.engine.api, Version=3.0.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.4.1\lib\nunit.engine.api.dll</HintPath>
+    <Reference Include="nunit.engine.api">
+      <HintPath>/usr/lib/mono/nunit/nunit.engine.api.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="System" />
