Index: xamarin-pkg-monodevelop/src/addins/AspNet/Properties/MonoDevelop.AspNet.addin.xml
===================================================================
--- xamarin-pkg-monodevelop.orig/src/addins/AspNet/Properties/MonoDevelop.AspNet.addin.xml	2015-04-28 10:16:13.348260401 +0100
+++ xamarin-pkg-monodevelop/src/addins/AspNet/Properties/MonoDevelop.AspNet.addin.xml	2015-04-28 10:16:13.344260401 +0100
@@ -1,12 +1,14 @@
 <ExtensionModel>
 	
 	<Runtime>
+<!--
 		<Import assembly = "System.Web.Helpers.dll" />
 		<Import assembly = "System.Web.Mvc.dll" />
 		<Import assembly = "System.Web.Razor.dll" />
 		<Import assembly = "System.Web.WebPages.Deployment.dll" />
 		<Import assembly = "System.Web.WebPages.Razor.dll" />
 		<Import assembly = "System.Web.WebPages.dll" />
+-->
 
 		<Import file = "Html/Schemas/xhtml1-frameset.xsd" />
 		<Import file = "Html/Schemas/xhtml1-strict.xsd" />
@@ -182,10 +184,12 @@
 			guid="{E53F8FEA-EAE0-44A6-8774-FFD645390401}"
 			type="MonoDevelop.AspNet.Projects.AspMvc3ProjectFlavor"
 			alias="AspNetMvc3" />
+<!--
 		<ProjectFlavor
 			guid="{E3E379DF-F4C6-4180-9B81-6769533ABE47}"
 			type="MonoDevelop.AspNet.Projects.AspMvc4ProjectFlavor" 
 			alias="AspNetMvc4" />
+-->
 		<ProjectFlavor
 			id="MonoDevelop.AspNet.Projects.AspNetAppProjectFlavor"
 			guid="{349C5851-65DF-11DA-9384-00065B846F21}"
Index: xamarin-pkg-monodevelop/src/addins/AspNet/MonoDevelop.AspNet.csproj
===================================================================
--- xamarin-pkg-monodevelop.orig/src/addins/AspNet/MonoDevelop.AspNet.csproj	2015-04-28 10:06:54.100238221 +0100
+++ xamarin-pkg-monodevelop/src/addins/AspNet/MonoDevelop.AspNet.csproj	2015-04-28 10:16:51.272261905 +0100
@@ -495,34 +495,6 @@
     <None Include="Templates\MvcCommon\ViewPageRazor.cshtml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.Mvc.5.2.3\lib\net45\System.Web.Mvc.dll">
-      <Link>System.Web.Mvc.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.Razor.3.2.3\lib\net45\System.Web.Razor.dll">
-      <Link>System.Web.Razor.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.Helpers.dll">
-      <Link>System.Web.Helpers.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.WebPages.Deployment.dll">
-      <Link>System.Web.WebPages.Deployment.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.WebPages.dll">
-      <Link>System.Web.WebPages.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.WebPages.Razor.dll">
-      <Link>System.Web.WebPages.Razor.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\..\packages\Microsoft.Web.Infrastructure.1.0.0.0\lib\net40\Microsoft.Web.Infrastructure.dll">
-      <Link>Microsoft.Web.Infrastructure.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
     <None Include="packages.config" />
     <None Include="Templates\Projects\MvcProject.xpt.xml">
       <SubType>Designer</SubType>
