--- tests/TestRunner/TestRunner.csproj.old
+++ tests/TestRunner/TestRunner.csproj
@@ -33,18 +33,10 @@
     <Reference Include="System" />
     <Reference Include="gtk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
     <Reference Include="System.Core" />
-    <Reference Include="nunit-console-runner">
-      <HintPath>..\..\external\mdtestharness\lib\nunit-console-runner.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core">
-      <HintPath>..\..\external\mdtestharness\lib\nunit.core.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core.interfaces">
-      <HintPath>..\..\external\mdtestharness\lib\nunit.core.interfaces.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\external\mdtestharness\nunit.framework.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit-console-runner"/>
+    <Reference Include="nunit.core"/>
+    <Reference Include="nunit.core.interfaces"/>
+    <Reference Include="nunit.framework"/>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />

--- src/addins/NUnit/NUnitRunner/NUnitRunner.csproj.old
+++ src/addins/NUnit/NUnitRunner/NUnitRunner.csproj
@@ -37,15 +37,9 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Xml" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\..\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core">
-      <HintPath>..\..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core.interfaces">
-      <HintPath>..\..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.interfaces.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit.framework"/>
+    <Reference Include="nunit.core"/>
+    <Reference Include="nunit.core.interfaces"/>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="NUnitTestRunner.cs" />
