--- a/Main.sln	2016-03-01 09:30:26.034007880 +0100
+++ b/Main.sln	2016-03-01 09:30:49.934006763 +0100
@@ -290,8 +290,6 @@
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WindowsPlatform.Tests", "tests\WindowsPlatform.Tests\WindowsPlatform.Tests.csproj", "{865100E2-A29C-4FCD-B803-1A0B9A0A6EF7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RefactoringEssentials", "external\RefactoringEssentials\RefactoringEssentials\RefactoringEssentials.csproj", "{C465A5DC-AD28-49A2-89C0-F81838814A7E}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MonoDevelop.CSharpBinding.Tests", "tests\MonoDevelop.CSharpBinding.Tests\MonoDevelop.CSharpBinding.Tests.csproj", "{84E22F5F-0562-4B51-8B9A-FD79FF33A8B5}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MonoDevelop.Projects.Formats.MSBuild.dotnet.v14.0", "src\core\MonoDevelop.Projects.Formats.MSBuild\MonoDevelop.Projects.Formats.MSBuild.dotnet.v14.0.csproj", "{8EE5A588-9445-48EC-97BD-BF066E7FFD51}"
@@ -300,8 +298,6 @@
 EndProject
 Project("{9344BDBB-3E7F-41FC-A0DD-8665D75EE146}") = "po", "po\po.mdproj", "{9E061B40-12E7-470A-B225-E566BAF745EB}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NUnit3Runner", "src\addins\MonoDevelop.UnitTesting.NUnit\NUnit3Runner\NUnit3Runner.csproj", "{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}"
-EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -1870,26 +1866,6 @@
 		{FEC19BDA-4904-4005-8C09-68E82E8BEF6A}.ReleaseMac|Any CPU.Build.0 = Release|Any CPU
 		{FEC19BDA-4904-4005-8C09-68E82E8BEF6A}.ReleaseWin32|Any CPU.ActiveCfg = Release|Any CPU
 		{FEC19BDA-4904-4005-8C09-68E82E8BEF6A}.ReleaseWin32|Any CPU.Build.0 = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugMac|Any CPU.ActiveCfg = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugMac|Any CPU.Build.0 = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugWin32|Any CPU.ActiveCfg = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugWin32|Any CPU.Build.0 = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Release|Any CPU.Build.0 = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugGnome|Any CPU.ActiveCfg = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.DebugGnome|Any CPU.Build.0 = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseMac|Any CPU.ActiveCfg = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseMac|Any CPU.Build.0 = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseWin32|Any CPU.ActiveCfg = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseWin32|Any CPU.Build.0 = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseGnome|Any CPU.ActiveCfg = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.ReleaseGnome|Any CPU.Build.0 = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Debug|x86.Build.0 = Debug|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Release|x86.ActiveCfg = Release|Any CPU
-		{C465A5DC-AD28-49A2-89C0-F81838814A7E}.Release|x86.Build.0 = Release|Any CPU
 		{84E22F5F-0562-4B51-8B9A-FD79FF33A8B5}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{84E22F5F-0562-4B51-8B9A-FD79FF33A8B5}.DebugMac|Any CPU.ActiveCfg = Debug|Any CPU
 		{84E22F5F-0562-4B51-8B9A-FD79FF33A8B5}.DebugWin32|Any CPU.ActiveCfg = Debug|Any CPU
@@ -1953,26 +1929,6 @@
 		{9E061B40-12E7-470A-B225-E566BAF745EB}.ReleaseMac|Any CPU.ActiveCfg = ReleaseMac|Any CPU
 		{9E061B40-12E7-470A-B225-E566BAF745EB}.ReleaseWin32|Any CPU.ActiveCfg = ReleaseWin32|Any CPU
 		{9E061B40-12E7-470A-B225-E566BAF745EB}.ReleaseGnome|Any CPU.ActiveCfg = ReleaseGnome|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugMac|Any CPU.ActiveCfg = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugMac|Any CPU.Build.0 = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugWin32|Any CPU.ActiveCfg = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugWin32|Any CPU.Build.0 = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Release|Any CPU.Build.0 = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugGnome|Any CPU.ActiveCfg = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.DebugGnome|Any CPU.Build.0 = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseMac|Any CPU.ActiveCfg = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseMac|Any CPU.Build.0 = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseWin32|Any CPU.ActiveCfg = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseWin32|Any CPU.Build.0 = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseGnome|Any CPU.ActiveCfg = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.ReleaseGnome|Any CPU.Build.0 = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Debug|x86.Build.0 = Debug|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Release|x86.ActiveCfg = Release|Any CPU
-		{D2A4E99E-FC2D-45A9-8BE7-1AB7DF95BA2A}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
 		{7525BB88-6142-4A26-93B9-A30C6983390A} = {9D360D43-0C05-49D6-84DB-4E7AB2F38F82}
