--- a/tests/TestRunner/TestRunner.csproj
+++ b/tests/TestRunner/TestRunner.csproj
@@ -33,18 +33,10 @@
     <Reference Include="System" />
     <Reference Include="gtk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
     <Reference Include="System.Core" />
-    <Reference Include="nunit-console-runner">
-      <HintPath>..\..\external\mdtestharness\lib\nunit-console-runner.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core">
-      <HintPath>..\..\external\mdtestharness\lib\nunit.core.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core.interfaces">
-      <HintPath>..\..\external\mdtestharness\lib\nunit.core.interfaces.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\external\mdtestharness\nunit.framework.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit-console-runner"/>
+    <Reference Include="nunit.core"/>
+    <Reference Include="nunit.core.interfaces"/>
+    <Reference Include="nunit.framework"/>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />

--- a/src/addins/NUnit/NUnitRunner/NUnitRunner.csproj
+++ b/src/addins/NUnit/NUnitRunner/NUnitRunner.csproj
@@ -37,15 +37,9 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Xml" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\..\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core">
-      <HintPath>..\..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core.interfaces">
-      <HintPath>..\..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.interfaces.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit.framework"/>
+    <Reference Include="nunit.core"/>
+    <Reference Include="nunit.core.interfaces"/>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="NUnitTestRunner.cs" />

--- a/src/addins/NUnit/MonoDevelop.NUnit.csproj
+++ b/src/addins/NUnit/MonoDevelop.NUnit.csproj
@@ -63,18 +63,10 @@
     <Reference Include="Mono.Cairo" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Core" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core">
-      <HintPath>..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.core.interfaces">
-      <HintPath>..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.core.interfaces.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.util">
-      <HintPath>..\..\..\packages\NUnit.Runners.2.6.3\tools\lib\nunit.util.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit.framework"/>
+    <Reference Include="nunit.core"/>
+    <Reference Include="nunit.core.interfaces"/>
+    <Reference Include="nunit.util"/>
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\core\MonoDevelop.Core\MonoDevelop.Core.csproj">


--- a/src/addins/NUnit/NUnitRunner/NUnitTestRunner.cs
+++ b/src/addins/NUnit/NUnitRunner/NUnitTestRunner.cs
@@ -102,7 +102,7 @@
 			if (!string.IsNullOrEmpty (suiteName))
 				package.TestName = suiteName;
 			tr.Load (package);
-			return tr.Run (listener, filter, false, LoggingThreshold.All);
+			return tr.Run (listener, filter);
 		}
 		
 		public NunitTestInfo GetTestInfo (string path, List<string> supportAssemblies)


--- a/external/nrefactory/ICSharpCode.NRefactory.Tests/ICSharpCode.NRefactory.Tests.csproj
+++ b/external/nrefactory/ICSharpCode.NRefactory.Tests/ICSharpCode.NRefactory.Tests.csproj
@@ -86,9 +86,7 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\cecil\Test\libs\nunit-2.5.10\nunit.framework.dll</HintPath>
-    </Reference>
+    <Reference Include="nunit.framework, Version=2.5.10.0, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\ICSharpCode.NRefactory\Properties\GlobalAssemblyInfo.cs">

