diff -up mono-4.2.1/mcs/build/library.make.than mono-4.2.1/mcs/build/library.make
diff -up mono-4.2.1/mcs/class/corlib/System/TimeZoneInfo.cs.than mono-4.2.1/mcs/class/corlib/System/TimeZoneInfo.cs
--- mono-4.2.1/mcs/class/corlib/System/TimeZoneInfo.cs.than	2015-11-12 04:00:29.000000000 -0500
+++ mono-4.2.1/mcs/class/corlib/System/TimeZoneInfo.cs	2016-01-18 10:28:29.532781004 -0500
@@ -1150,7 +1150,7 @@ namespace System
 			try {
 				return ParseTZBuffer (id, buffer, length);
 			} catch (Exception e) {
-				throw new InvalidTimeZoneException (e.Message);
+				return Utc; //throw new InvalidTimeZoneException (e.Message);
 			}
 		}
 
