--- a/mcs/build/executable.make	2016-08-23 17:03:34.146852360 +0200
+++ b/mcs/build/executable.make	2016-08-23 17:04:30.926229173 +0200
@@ -113,7 +113,7 @@
 $(the_lib): $(the_libdir)/.stamp
 
 $(build_lib): $(BUILT_SOURCES) $(EXTRA_SOURCES) $(response) $(build_libdir:=/.stamp)
-	$(PROGRAM_COMPILE) $(MCS_REFERENCES) -target:exe -out:$@ $(BUILT_SOURCES) $(EXTRA_SOURCES) @$(response)
+	$(PROGRAM_COMPILE) $(MCS_REFERENCES) $(LIB_MCS_FLAGS) -target:exe -out:$@ $(BUILT_SOURCES) $(EXTRA_SOURCES) @$(response)
 ifdef PROGRAM_SNK
 	$(Q) $(SN) -R $@ $(PROGRAM_SNK)
 endif
